core-banking-lib/src/main/java/at/deloittedigital/core_banking/lib/CoreBankingLibApplication.java

```diff
@@ -0,0 +1,13 @@
+package at.deloittedigital.core_banking.lib;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CoreBankingLibApplication {
```

Comment:
What is the reason for having an application definition in the library project?

Reply:
Forgot to remove it after creating the library with Spring Initializr, thank you.